### DISCUSSION TITLE
How To Get The Previous Tracking Settings?

### DISCUSSION THREAD

**Discussion by sonerkizilgol** [alvr-org/ALVR#1250] [Q&A]
Both Oculus and Fast prediction settings worked very well for me. However, since the last update turned those settings into a slider in the advanced settings, I am not sure how to use them anymore to get the same results. Can someone explain to me what each advanced controller setting does because I have no idea.

**Comment by sonerkizilgol** [comment]
Update: I still think these settings aren't explained properly, but setting the clientsideControllerPredictionMultiplier to 1 and setting 
both steamvrHmdPredictionMultiplier and steamvrCtrlPredictionMultiplier to 0 reduced the perceived input lag tremendously. Maybe a bit more shaky but at least I don't feel like I'm living in the past.

**Reply by zmerp** [reply]
Are you on windows or linux?

**Reply by sonerkizilgol** [reply]
I'm on Windows
